change the exclude from dayjs

xopez opened the pull request:
Since en (https://raw.githubusercontent.com/cdnjs/cdnjs/master/ajax/libs/dayjs/1.10.4/locale/en.js) is available and en-us not (HTTP 404) we should just switch the condition.
Also we should use the  `min.js` files. The files are available for every locale file. It saves about a byte because the non minimized file has a linebreak (LF). Also some variables got replaced. It doesn't save much, but a little bit.

xopez committed:
change the exclude from dayjs

Since en (https://raw.githubusercontent.com/cdnjs/cdnjs/master/ajax/libs/dayjs/1.10.4/locale/en.js) is available and en-us not (HTTP 404) we should just switch the condition.
Also we should use the  `min.js` files. The files are available for every locale file. It saves about a byte because the non minimized file has a linebreak (LF). Also some variables got replaced. It doesn't save much, but a little bit.

xopez commented:
Diff of german locales:
![grafik](https://user-images.githubusercontent.com/28950736/106585244-68c8b700-6547-11eb-8cea-08ae440fb6e0.png)

Wruczek commented:
The if statement is correct. The language code used for locale file is different than the ts-website language code, and comes from `DAYJS_LANG` variable set in every language.
The `DAYJS_LANG` for English (US) is actually just `en`, and it's the default language already bundled with day.js - that's why we don't need to load it.
The `DAYJS_LANG` for English is `en-gb`, and we need to load it from the CDN.

xopez committed:
Revert

xopez committed:
Update dbinstall_mysql_lang.sql

xopez commented:
My bad. I reverted te condition. I forgot to change that 2 lines in my database: 
https://github.com/Wruczek/ts-website/commit/c751001c6182caaacb0a4592f285a6878fd6d5ff#diff-7229ab00f5539e46dcb2d0c1c51fab463f201a3b09580b48f52216bedddfce22L80-R81

I changed my PR now. I also changed that in the table `languages`.

Wruczek commented on src/installer/dbinstall_mysql_lang.sql:
```diff
@@ -19,8 +19,8 @@ CREATE TABLE `DBPREFIXlanguages` (
 INSERT INTO `DBPREFIXlanguages` (`langid`, `englishname`, `nativename`, `langcode`, `isdefault`) VALUES
-(1, 'English', 'English', 'en', 1),
-(2, 'English (US)', 'English (US)', 'en-us', 0),
+(1, 'English', 'English', 'en-gb', 1),
```
Again: this is not the language used in day.js
The language for day.js is pulled from `DAYJS_LANG` variable set in every language.

xopez replied:
Shouldn't we set this accordingly like the `DAYJS_LANG`? That is what I thought about that change. Anyway, I will remove it for now

Wruczek replied:
As far as I remember, those are the language codes from our translation platform.

xopez committed:
Update dbinstall_mysql_lang.sql
